docs/pages/installation.mdx

```diff
@@ -48,56 +49,345 @@ supports most features on Windows 10 and later.*
 
 ## Linux
 
-All installations include `teleport`, `tsh`, `tctl`, and `tbot`.
+This section shows you how to install Teleport binaries on a single Linux
+server.
 
-### Feature support
+Teleport maintains DEB and RPM package repositories for different operating
+systems, platforms, and Teleport versions. A server that installs Teleport from
+a DEB or RPM package must have systemd installed. You can also download TAR
+archives containing Teleport binaries. All installations include `teleport`,
+`tsh`, `tctl`, and `tbot`.
 
-Some Teleport features have additional requirements:
+### Recommended installation steps
 
-| Feature                                                                        | Requirement   | Debian                   | Ubuntu   | CentOS/RHEL | Amazon Linux           | SLES           |
-|--------------------------------------------------------------------------------|---------------|--------------------------|----------|-------------|------------------------|----------------|
-| [Enhanced Session Recording](./server-access/guides/bpf-session-recording.mdx) | Kernel  v5.8+ | 11, or 10 with backports | 20.04.2+ | 9+          | 2 (post 11/2021), 2023 | 12 SP5, 15 SP5 |
-| Automatic Updates                                                              | systemd-based | 9+                       | 16.04+   | 7+          | 2, 2023                | 12 SP5, 15 SP5 |
-| Installation through apt/yum/zypper repos                                      | systemd-based | 9+                       | 16.04+   | 7+          | 2, 2023                | 12 SP5, 15 SP5 |
+If you are starting out with Teleport, we recommend beginning with a [Teleport
+Team](https://goteleport.com/signup/) account. From there, the only Teleport
```

Comment:
Nit: I would say "Teleport Cloud" instead of "Teleport Team" to keep it more generic. We're introducing other plans soon.

```suggestion
Cloud](https://goteleport.com/signup/) account. From there, the only Teleport
```